README.md

```diff
@@ -1 +1,111 @@
-# webpacktodolist
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+<img src="./microverse.png" alt="logo" width="140"  height="auto" />
+<br/>
+
+<h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+  - [🚀 Live Demo](https://pierrebarba.github.io/finishProjectCapton/index.html)
+- [💻 Getting Started](https://pierrebarba.github.io/finishProjectCapton/index.html)
+  - [Setup](setup)
+  - [Prerequisites](https://pierrebarba.github.io/finishProjectCapton/index.html)
+  - [Install](install)
+  - [Usage](html,javascript,css)
+- [👥 Authors](https://github.com/PierreBarba)
+- [🔭 Future Features](https://pierrebarba.github.io/finishProjectCapton/index.html)
+- [🤝 Contributing](https://github.com/cilfonegabriel)
+- [⭐️ Show your support](support)
+- [🙏 Acknowledgements](https://github.com/elafildecolor,https://github.com/luigirazum)
+- [📝 License](MIT)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To Do List] <a name="Todo List"></a>
```

Comment:
Good job implementing some of the changes requested by the previous code reviewer you have added a project title but you forgot to add a project description it would be nice if you added a project description to your project tell us what the project your are building is about.

Reply:
thanks  you I will do that have a great day. 😄